packages/cli/src/databases/entities/execution-entity.ts

```diff
@@ -47,6 +47,13 @@ export class ExecutionEntity {
 	status: ExecutionStatus;
 
 	@Column(datetimeColumnType)
+	createdAt: Date;
+
+	/**
+	 * Time when the processing of the execution actually started. This column
+	 * is `null` when an execution is enqueued but has not started yet.
+	 */
+	@Column({ type: datetimeColumnType, nullable: true })
 	startedAt: Date;
```

Comment:
The type is now wrong, should be `Date | null`, right?

Reply:
This seems to cause type issues elsewhere, checking.

Reply:
Fixed types here: [7c857f1](https://github.com/n8n-io/n8n/pull/10810/commits/7c857f184c4c0656515686f715f3014eeb7d49fb)

Our execution types need heavy simplification, hopefully good enough for our purposes here now.

Reply:
That's probably true, since we can't anymore assume it's always defined, like it has been now